temporal-sdk/src/main/java/io/temporal/internal/worker/PollerOptions.java

```diff
@@ -48,6 +48,8 @@ public static PollerOptions getDefaultInstance() {
 
     private int maximumPollRateIntervalMilliseconds = 1000;
 
+    private int taskExecutorThreadPoolSize;
```

Comment:
I don't think poller options is the right place for the executor thread pool size.

Reply:
We need to access it in the poller, so it's either this or we should be passing more parameters.